lib/commands/init.js

```diff
@@ -81,7 +81,7 @@ const initFunction = async () => {
 
     fs.mkdirSync(functionDir, "777");
 
-    let gitInitCommands = "git clone --depth 1 --sparse https://github.com/appwrite/functions-starter ."; // depth prevents fetching older commits reducing the amount fetched
+    let gitInitCommands = "git clone -b v3 --single-branch --depth 1 --sparse https://github.com/appwrite/functions-starter ."; // depth prevents fetching older commits reducing the amount fetched
```

Comment:
This repo doesn't work, does it?

Reply:
hmm it does ? https://github.com/appwrite/functions-starter/releases/tag/v3